pkg/loader/compose/compose.go

```diff
@@ -337,7 +337,12 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 		if composeServiceConfig.Volumes != nil {
 			for _, volume := range composeServiceConfig.Volumes.Volumes {
 				v := normalizeServiceNames(volume.String())
-				serviceConfig.Volumes = append(serviceConfig.Volumes, v)
+				if strings.Contains(v, ".") {
```

Comment:
Instead of putting this if statement here (making `compose.go` even larger). Why not put the if / else statement inside of `func normalizeVolumeNames` ?